tests/jballerina-unit-test/src/test/resources/test-src/statements/dostatement/do-stmt.bal

```diff
@@ -15,6 +15,12 @@ function testOnFailStatement() {
     string returnWithinOnFailResult = testReturnWithinOnFail();
     assertEquality("Before failure throw-> Error caught !", returnWithinOnFailResult);
 
+    string resultOnFailWithoutVariable = testOnFailWithoutVariable();
+    assertEquality("Before failure throw-> Error caught !", resultOnFailWithoutVariable);
+
+    string resultWithoutExeOnFailWithoutVariable = testWithoutExeOnFailWithoutVariable();
```

Comment:
Can we improve this to say something like `testOnFailWithoutVariableWithoutOnFailExecution`?